bentoml/server/api_server.py

```diff
@@ -454,6 +454,7 @@ def handle_ml_processing(self):
         logger.info('requests status_code for GET subscriptionQuery: %s', r.status_code)
         logger.info('Data: %s', r.json())
         ENTITY_INPUT_DATA = r.json()['entityID']['value']
+        ATTRIBUTE_INPUT_DATA = r.json()['query']['value'].split('=')[1]
```

Comment:
I don't get this.

The watched attributes that will be used as inputs to the ML model are defined by the ML Model entity, not by the subscription query.

Reply:
I'm confused. What is/should be in the subscriptionQuery entity if not the input data desired by the consumer app ? If the watched attributes are defined by the ML Model entity, what is the purpose of the subscriptionQuery ?

Reply:
The ML Model declares what it accepts as input data (e.g. a `precipitation` property), the subscription query says where this property will be found (e.g. in a specific `River` entity)